exercises/basic_reduce/solution/solution.js

```diff
@@ -1,8 +1,8 @@
 function countWords(arr) {
-  return arr.reduce(function(countMap, word) {
-    countMap[word] = ++countMap[word] || 1 // increment or initialize to 1
-    return countMap
-  }, {}) // second argument to reduce initialises countMap to {}
+  return arr.reduce(function(accumulator, current_word) {
```

Comment:
The original argument names were intended to be "semantic", related to the problem, rather than generic "accumulator". i.e. you're creating a `map` of `counts` for `words`.

Reduce functions can be hard to follow, so the idea is to embed as much semantic information into the argument names as possible. I'm open to better names, so long as they're not generic.